packages/discovery-provider/src/tasks/index_user_bank.py

```diff
@@ -258,6 +259,16 @@ def process_create_userbank_instruction(
         logger.error(e)
 
 
+def get_transfer_type_from_memo(memos: List[str]) -> USDCTransactionType:
+    """Checks the list of memos for one matching containing a transaction type and returns it if found. Defaults to USDCTransactionType.transfer if no matching memo is found"""
+    for memo in memos:
+        if memo == "prepareWithdrawal":
```

Comment:
Super nit, but might look nice in block explorers to have these be more "human" - eg. "Prepare Withdrawal"

Reply:
Sounds good, will update.

Reply:
Done!